docker-compose.platform.yml

```diff
@@ -6,7 +6,7 @@ services:
     restart: unless-stopped
     volumes:
       - drive_mongodb_data:/data/db
-      - ./docker/mongodb/initiate_mongodb_replica.js:/docker-entrypoint-initdb.d/initiate_mongodb_replica.js
+      - ~/.mn/${CONFIG:?err}/initiate_mongodb_replica.js:/docker-entrypoint-initdb.d/initiate_mongodb_replica.js
     extra_hosts:
       - "drive_mongodb:127.0.0.1"
     command: mongod --replSet driveDocumentIndices --bind_ip_all
```

Comment:
We need to pass MongoDB port

Reply:
I think that is done here?
https://github.com/dashevo/mn-bootstrap/pull/150/files/602e2bbb9cdba4e4457f9fa2c7cfa47f7fc3c8c8#diff-399795f6a9dce5cdec6c156808e7894827be79918aff4b7674b2249a550a49f9R8

Reply:
Nope, there is how we connect to a replica node but not what port the node is listening on. We need to pass `--port` option to `mongod`.

Reply:
I tried putting mongodb on a configurable port (e.g. 27027) and repeatedly ran into this error:

```
/usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/initiate_mongodb_replica.js
2020-10-14T03:04:31.192+0000 I  NETWORK  [listener] connection accepted from 127.0.0.1:33712 #2 (1 connection now open)
2020-10-14T03:04:31.192+0000 I  NETWORK  [conn2] received client metadata from 127.0.0.1:33712 conn2: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "4.2.10" }, os: { type: "Linux", name: "Ubuntu", architecture: "x86_64", version: "18.04" } }
2020-10-14T03:04:31.220+0000 I  REPL     [conn2] replSetInitiate admin command received from client
2020-10-14T03:04:31.221+0000 E  REPL     [conn2] replSet initiate got NodeNotFound: No host described in new configuration 1 for replica set driveDocumentIndices maps to this node while validating { _id: "driveDocumentIndices", version: 1.0, members: [ { _id: 0.0, host: "drive_mongodb:27027" } ] }
2020-10-14T03:04:31.243+0000 I  NETWORK  [conn2] end connection 127.0.0.1:33712 (0 connections now open)
```

This made me realise that we don't actually need to add most of these ports to the config because they are internal to the named network, right? Since each network has its own namespace, they can use the same/default ports internally, only published ports need to be configurable so they don't conflict on the host. I think the list of published ports is:
- dashd P2P
- dashd RPC (host only, for status command)
- tendermint P2P
- nginx jsonrpc
- nginx grpc

Is my thinking right, can we remove a lot of this from config again?

Reply:
Yeah, you right.

Reply:
OK, I cleaned up all config except relating to these 5 ports.